blog/_posts/2022-09-29-scala-survey-2022-announcement.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: blog-detail
+post-type: blog
+by: Kacper Korban, VirtusLab
+title: Participate in the Scala Survey 2022!
+---
+
+![Scala Survey 2022 Announcement]({{ site.baseurl }}/resources/img/scala-survey-2022-announcement.png)
+
+## The Scala Team wants to hear from you!
+
+We want to make the language we all love as amazing as it possibly can be. We want to know which way the Scala community is moving and which parts of the Scala language you utilize the most. That’s why we are asking you to help by taking part in the Scala Survey 2022.
```

Comment:
How about making `Scala Survey 2022` here a link instead of adding it below?

Reply:
I thought about it, but the whole point of this post is to give the link to the survey. And this way it is more explicit and more visible.